### DISCUSSION TITLE
Welcome to Onepanel discussions!

### DISCUSSION THREAD

**Discussion by rushtehrani** [onepanelio/core#665] [Announcements]
## 👋  Welcome!

In addition to Slack, we are also using Discussions as a place to connect with members of our community. We hope that you:

  - Ask questions and get help.
  - Share ideas and feature requests.
  - Engage with other community members.
  - Welcome others and are open-minded.

To get started, comment below with an introduction of yourself and tell us about what you do or hope to achieve with this community.

**Comment by aleksandrmelnikov** [comment]
My name is Aleksandr "Alex" Melnikov.

I am one of the founding engineers for Onepanel.

I help build Onepanel and provide support for Onepanel related issues.

**Comment by Vafilor** [comment]
Hi everyone,

My name is Andrey Melnikov.

I am also one of the founding engineers for Onepanel.

Apart from contributing to the project, I help manage it to make sure we're hitting our release goals.
